src/resources/views/crud/fields/relationship/inline_create_button.blade.php

```diff
@@ -0,0 +1,3 @@
+<button type="button" class="btn btn-sm btn-link float-right" id="{{ $field['inline_create']['entity'] }}-inline-create-{{$field['name']}}"><span class="la la-plus"></span>{{trans('backpack::crud.add')}}</span></button>
```

Comment:
## Relationship Issue 13

When clicking the ```inline_create_button```, the overlay is VERY dark for a second, then gets to a lighter shade of dark (which is better). This darker flicker is a small thing, but something that makes it look a kind of buggy:

![2020-03-15 16 48 18](https://user-images.githubusercontent.com/1032474/76703793-feafe500-66dc-11ea-93e7-76e2be24968e.gif)

This would look MUCH better if it weren't for that dark flicker. Anything we can do about it @pxpm ?

Reply:
@pxpm Reminder: don't forget about this flicker too.